proto/node.proto

```diff
@@ -130,3 +131,11 @@ message SwapResponse {
   repeated bdhke.BlindSignature signatures = 1;
 }
 
+message AckRequest {
+  string method = 1;
+  string request_hash = 2;
+}
+
+message AckResponse {
+  string status = 1;
```

Comment:
I don't even think the client should expect a response.
He could just fire and forget. Doesn't change a thing for him.
In the worst case, the node will end up pruning later instead of right now.

So I think:
`message AcknowledgeResponse {}` 
is enough.